pkg/common/ingress/controller/controller.go

```diff
@@ -196,7 +196,7 @@ func (ic GenericController) GetFullResourceName(name, currentNamespace string) s
 		return ""
 	}
 	if strings.Index(name, "/") == -1 {
-		// there isn't a slash, just the resourcename
+		// there isn't a slash, just the resource name
```

Comment:
Only esthetic changes, please remove all the changes on this file from the commit.